### PR TITLE
Add information about the examples to the main readme

### PR DESCRIPTION
**Pull request by teemupiiroinenwirepas**
Signed-off-by: Teemu Piiroinen <teemu.piiroinen@wirepas.com>

### CODE REVIEW

File: wrappers/python/README.md [hunk 1]
```diff
@@ -47,6 +47,16 @@ For development mode installation use
     pip install -e .
 ```
 
+You can also install the package from the dist folder with the examples
```
Comment (PFigs) [inline wrappers/python/README.md]: Please rewrite the sentence so it is clear that the [examples] will install the extra requirements needed by the example code (in this case wnt examples)

Comment (teemupiiroinenwirepas) [reply wrappers/python/README.md]: Done

File: wrappers/python/README.md [hunk 2]
```diff
@@ -47,6 +47,16 @@ For development mode installation use
     pip install -e .
 ```
 
+For installing also the requirements needed to run the examples
```
Comment (PFigs) [inline wrappers/python/README.md]: Much better. I would drop the also, I don't think it is needed.

Comment (teemupiiroinenwirepas) [reply wrappers/python/README.md]: Done

**Pull request merged**